webapp/components/tokenSelector/tokenList.tsx

```diff
@@ -61,6 +61,7 @@ const List = function ({
         rows.push(
           <div
             className="absolute left-0 top-0 w-full"
+            key="manually_added"
```

Comment:
Will we have only one item with this key?

Reply:
this is part of the array a few lines below. There, each element of the array has a unique key. If I don't add the key here, it throws a warning, so it is required.